package.json

```diff
@@ -13,7 +14,7 @@
     "clean": "rm -rf dist",
     "lint": "github-lint",
     "prebuild": "npm run clean && npm run lint && mkdir dist",
-    "build": "rollup -c && cp index.js.flow dist/index.esm.js.flow && cp index.js.flow dist/index.umd.js.flow",
+    "build": "rollup -c && cp index.js.flow dist/index.esm.js.flow && cp index.js.flow dist/index.umd.js.flow && cp index.d.ts dist/index.d.ts",
```

Comment:
What do you think about leaving the file in the root and adding it to the `files` key above?

Reply:
Sure!